tools/jenkins/openshift/jobs-config.yaml

```diff
@@ -1,9 +1,10 @@
 ---
 kind: Template
 apiVersion: v1
+labels:
```

Comment:
Can we not achieve the same thing with -l app=jenkins-prod, like we do when processing other templates (ex. build-master.yaml)? Otherwise, let's update all templates and remove that from the command lines.

Reply:
Good point - I'll do a quick pass on the remaining templates

Reply:
I think it's better to just update the readme instructions and rollback these templates.  I believe Clecio was labelling so much on the command line so Jenkins could build a Jenkins PR... not that we are doing that.  But something to consider.  One way or the other, as long as it's consistent.